### PR TITLE
Get endpoint uri test

### PR DESCRIPTION
**Pull request by adrianvrj**
Added two test cases for get_endpoint_uri function, in case an endpoint is found and in case it is not.

### CODE REVIEW

**Review by Gonmeso — CHANGES_REQUESTED** [review]
Thank you for your input!

Couple of things:

* You should update this branch with the latest changes in `main` as we have added testing in the CI
* Run `pre-commit` which would format and order the imports
* Add the data for the test inside the tests for clarity

File: tests/test_utils.py
[the remark this answers is not in the document]
Comment (adrianvrj) [reply tests/test_utils.py:1]: Already done this, you can see it at 08cf401 commit

File: tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from unittest.mock import *
+import pytest
+from giza.schemas.endpoints import Endpoint, EndpointsList
+from giza_actions.utils import get_endpoint_uri
+
+endpoint_data = Endpoint(id=999, size="S", is_active=True, model_id=999, version_id=999, uri="testing.uri")
+endpoint_list = EndpointsList(root=[endpoint_data])
```
[the remark this answers is not in the document]
Comment (adrianvrj) [reply tests/test_utils.py]: Already done this too, it is the df54f72 commit

**Comment by adrianvrj** [comment]
@Gonmeso I also informed the other devs in charge of this issue to make this corrections.

File: tests/test_utils.py
[the remark this answers is not in the document]
Comment (Gonmeso) [reply tests/test_utils.py:1]: Running pre-commit locally yields changes:

```bash
Check Yaml...........................................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for added large files..............................................Passed
Check for merge conflicts................................................Passed
Don't commit to branch...................................................Passed
pyupgrade................................................................Passed
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing /Users/gonmeso/src/external/actions-sdk/tests/test_utils.py

black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted tests/test_utils.py

All done! ✨ 🍰 ✨
1 file reformatted.

ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

tests/test_utils.py:1:1: F403 `from unittest.mock import *` used; unable to detect undefined names
tests/test_utils.py:3:8: F401 [*] `pytest` imported but unused
tests/test_utils.py:9:2: F405 `patch` may be undefined, or defined from star imports
tests/test_utils.py:29:2: F405 `patch` may be undefined, or defined from star imports
Found 4 errors.
[*] 1 fixable with the `--fix` option.
```

Run `pre-commit run --files tests/test_utils.py` and fix the formating and linting issues

File: tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from unittest.mock import *
```
Comment (Gonmeso) [inline tests/test_utils.py:1]: Importing with `* ` is considered a bad practice and usually discouraged in lintiing tools, only import what is needed, which is `patch`

File: tests/test_utils.py
[the remark this answers is not in the document]
Comment (adrianvrj) [reply tests/test_utils.py:1]: I think now I get it right

File: tests/test_utils.py (hunk 2 above)
Comment (adrianvrj) [reply tests/test_utils.py]: Fixed here d3bd146

File: tests/test_utils.py
[the remark this answers is not in the document]
Comment (adrianvrj) [reply tests/test_utils.py:1]: image

**Review by Gonmeso — CHANGES_REQUESTED** [review]
Two other tests have been added by your colleague, please update from main to add them and fix any conflict, then it will be good to go!

**Review by Gonmeso — APPROVED** [review]
LGTM!

**Pull request merged**